components/Button.js

```diff
@@ -23,11 +23,34 @@ const DefaultButton = styled.button`
     variant === "cancel" &&
     css`
       margin: 10px;
+      font-weight: 600;
+      align-self: flex-end;
     `}
 
   ${({ variant }) =>
     variant === "details" &&
     css`
       font-weight: 600;
+      font-size: 16px;
+    `}
+
+    ${({ variant }) =>
+    variant === "create" &&
+    css`
+      font-weight: 600;
+      font-size: 16px;
+      padding: 8px 65px 8px 65px;
```

Comment:
Just a tiny shorthand tipp 🤏
```suggestion
      padding: 8px 65px;
```